docs/docusaurus.config.ts

```diff
@@ -0,0 +1,157 @@
+import type * as Preset from '@docusaurus/preset-classic';
+
+// This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
+
+const config: Config = {
+  title: 'Renku',
+  tagline: 'Connecting data, code, compute, and people.',
+  favicon: 'img/favicon.ico',
+
+  // Future flags, see https://docusaurus.io/docs/api/docusaurus-config#future
+  future: {
+    v4: true, // Improve compatibility with the upcoming Docusaurus v4
+  },
+
+  // Set the production url of your site here
+  url: process.env.READTHEDOCS_CANONICAL_URL || "https://renku.readthedocs.io",
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: '/',
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: 'SwissDataScienceCenter', // Usually your GitHub org/user name.
+  projectName: 'renku', // Usually your repo name.
+
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+
+  markdown: { mermaid: true },
+
+  themes: [
+    '@docusaurus/theme-mermaid',
+  ],
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en'],
+  },
+
+  presets: [
+    [
+      'classic',
+      {
+        docs: {
+          sidebarPath: './sidebars.ts',
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            'https://github.com/SwissDataScienceCenter/renku/docs',
+        },
+        theme: {
+          customCss: './src/css/custom.css',
+        },
+      } satisfies Preset.Options,
+    ],
+  ],
+
+  themeConfig: {
+    // Replace with your project's social card
+    image: 'img/docusaurus-social-card.jpg',
+    navbar: {
+      title: 'Docs',
+      logo: {
+        alt: 'My Site Logo',
+        src: 'img/logo.svg',
+      },
+      items: [
+        {
+          type: 'docSidebar',
+          sidebarId: 'users',
+          position: 'left',
+          label: 'For users',
+        },
+        {
+          type: 'docSidebar',
+          sidebarId: 'admins',
+          position: 'left',
+          label: 'For admins',
+        },
+        {
+          href: 'https://renkulab.io',
+          label: 'Renkulab',
+          position: 'right',
+        },
+        {
+          href: 'https://github.com/SwissDataScienceCenter/renku',
+          label: 'GitHub',
+          position: 'right',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
+      links: [
+        {
+          title: 'Docs',
+          items: [
+            {
+              label: 'For users',
+              to: '/docs/users',
+            },
+            {
+              label: 'For admins',
+              to: '/docs/admins',
```

Comment:
Links to a 404 page, as the admin category doesn't have an index page.

This breaks the build step (returns a non-zero exit code).

```suggestion
              to: '/docs/admins/architecture/services',
```